front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -16,19 +16,23 @@ const slideInOutEasing = 'cubic-bezier(0.19, 1, 0.22, 1)';
 
 const Container = styled.div<{
   windowHeight: number;
+  windowWidth?: number;
   contentBgColor?: InputProps['contentBgColor'];
 }>`
-  width: 100vw;
-  height: ${({ windowHeight, theme }) =>
-    `calc(${windowHeight}px - ${theme.menuHeight}px)`};
   position: fixed;
-  top: ${({ theme }) => theme.menuHeight}px;
-  left: 0;
   display: flex;
   overflow: hidden;
   background: ${({ contentBgColor }) =>
     contentBgColor ? colors[contentBgColor] : colors.white};
-  z-index: 1003;
+  width: ${({ windowWidth }) => windowWidth}px; /* Fallback for width */
+  @supports (width: 100dvw) {
+    width: 100dvw; /* Dynamic viewport width for better mobile handling */
+  }
+  height: ${({ windowHeight }) => windowHeight}px; /* Fallback for height */
+  @supports (height: 100dvh) {
+    height: 100dvh; /* Dynamic viewport height for better mobile handling */
```

Comment:
This is good to know.. do you think we shuold be using `dvh/dvw` rather than `vh/vw` across the platform in our CSS then? :thinking: If so - maybe something to bring up at our next CDM? :)

Reply:
Good point. I'll add it to my list.